docs/spec/v1beta2/helmrepositories.md

```diff
@@ -1,9 +1,15 @@
 # Helm Repositories
 
-The `HelmRepository` API defines a Source to produce an Artifact for a Helm
-repository index YAML (`index.yaml`).
+There are 2 Helm repository types defines by the `HelmRepository` API: 
```

Comment:
I would change the order of the two entries, this aligns with the order below and follows historical order.